### PR TITLE
ci: split linting to seperate job

### PR DESCRIPTION
**Pull request by henryiii**
Pulls the linting out so it happens once and in parallel with the tests.

Also dropped `-ra` since it's already specified in `pyproject.toml`.

### CODE REVIEW

**Comment by jpivarski** [comment]
@ckoraka, let's not merge this because the tests aren't running, and the Coding Jam is about to begin. Even though this is probably an improvement, it's better to not rock the boat at this point.

**Comment by ckoraka** [comment]
@jpivarski I agree - I am quite curious though as to why the tests are not commencing. @henryiii is this behavior expected ?

**Comment by jpivarski** [comment]
Maybe it has nothing to do with the PR itself. I wonder if it's a setting in GitHub, something that doesn't trust outside PRs? I'd like to take a look at your screen...

**Comment by jpivarski** [comment]
To make things less confusing, I'm going to close this PR so that we don't see it in the list. However, there are good changes here. (We can always bring it back!)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -15,6 +15,23 @@ env:
   FORCE_COLOR: 3
 
 jobs:
+  name: Check code quality
+    runs-on: ubuntu-latest
```
Comment (henryiii) [inline .github/workflows/ci.yml:19]: Ahh, this is the typo. The indention can't change here.

**Pull request closed without merging**